### ISSUE TITLE
Create testing set (themes, videos, wheels, etc)

### ISSUE THREAD

**Issue by vikbez** [HFScode/zspin#9]
We need this to test everything is ok, and for supporting each different setting.
This must be as light as possible and redistribuable (copyright free)

**Comment by vikbez** [comment]
ftp://hyperfreespin.fr/HFCloud/Depot%20des%20medias/DemoBox.zip

**Comment by Odissine** [comment]
It seems there is an issue when we create a .zip file from OSX and uses it into a theme ... It's only works when I create the Zip file from windows :/ Maybe the hidden file ?

**Comment by vikbez** [comment]
Yes, the library used for extracting zip files on osx used in hyperspin does not handle zipfiles created with osx (see https://github.com/EvanOxfeld/node-unzip/issues/79)

however if you create the zipfile with command line `zip -r theme.zip folder/*` it works.
